features/shop/apply_correct_simple_promotion_on_order.feature

```diff
@@ -0,0 +1,32 @@
+@ui-cart
+Feature: Apply correct simple promotions on order
+    In order to pay proper amount while buying promoted goods
+    As a Customer
+    I want to have promotions applied
+
+    Background:
+        Given the store is operating on a single channel
+        And default currency is "EUR"
+        And store has a product "PHP T-Shirt" priced at "€100.00"
+        And store has a product "Symfony Mug" priced at "€5.00"
+        And there is promotion "Special discount"
+        And it has "€10.00" "fixed discount" for cart with "item total" above "€80.00"
```

Comment:
Can we use discount types without wrapping them with quotation marks? `(.+? discount) for cart`